tests/functional/simple_snapshot/test_hard_delete_snapshot.py

```diff
@@ -24,6 +36,12 @@ def datetime_snapshot():
     return begin_snapshot_datetime
 
 
+@pytest.fixture(scope="class", autouse=True)
+def setUp(project):
```

Comment:
was messing around in here and left this change as it better reflects our current best practices